asapdiscovery-alchemy/asapdiscovery/alchemy/schema/fec.py

```diff
@@ -15,7 +15,10 @@
 )
 from openfe.protocols.openmm_utils.omm_settings import (
     IntegratorSettings,
+    LambdaSettings,
```

Comment:
This is pre OFE 1.0. Must be a merge artifact,

Reply:
nice catch!